### DISCUSSION TITLE
Could not connect to Phoenix with JDK8

### DISCUSSION THREAD

**Discussion by Syehunter** [dbeaver/dbeaver#19704] [Q&A] [answered]
When using V22.0.4.202205011839 on Windows I couldn't connect to Phoenix with JDK8 because of DBeaver needs JDK at least 11

So that is the problem, if I set parameter like "-vm <jdk-path1.8>" the DBeaver can't even start, also it doesn't work on setting this in shell before connection

But if I don't set the parameter, emmmm......↓
class org.[apache](https://so.csdn.net/so/search?q=apache&spm=1001.2101.3001.7020).hadoop.security.JniBasedUnixGroupsMappingWithFallback not org.apache.hadoop.security.GroupMappingServiceProvider

How can I solve this problem?

The problem occurs with Phoenix5.0.0 && HBase2.1.0, yet it works well with Phoenix5.1.2 && HBase2.4.11(latest version)， I don't know why...

**Comment by serge-rider** [comment, marked as answer]
Similar to #15676
Try to use driver 4.14.1-HBase-1.4.
New DBeaver versions won't support legacy JDK (it is too old).

Also you could try to use an old version of DBeaver (6.x I guess).
